test/inductor/test_flex_attention.py

```diff
@@ -1331,6 +1331,30 @@ def test_captured_score_mod_aot_eager_gradcheck(
             )
         )
 
+    @supported_platform
+    @common_utils.parametrize("mode", ["eager", "aot_eager"])
+    def test_document_masking_edge_case(self, mode):
```

Comment:
cc @Chillee  verified this test would have caught, is that good?

Reply:
oh I see, it's running in 'eager' mode. Makes sense.